tests/components/hardware/test_websocket_api.py

```diff
@@ -16,3 +20,51 @@ async def test_board_info(hass: HomeAssistant, hass_ws_client) -> None:
     assert msg["id"] == 1
     assert msg["success"]
     assert msg["result"] == {"hardware": []}
+
+
+TEST_TIME_ADVANCE_INTERVAL = datetime.timedelta(seconds=5 + 1)
+
+
+async def test_mqtt_ws_subscription(hass: HomeAssistant, hass_ws_client, freezer):
```

Comment:
Should this be mqtt? or copy paste that was missed

Reply:
it's indeed a copy-paste that was missed :)